addons/test_sale_product_configurators/tests/test_sale_product_configurator.py

```diff
@@ -87,12 +86,10 @@ def test_02_product_configurator_advanced(self):
 
         self.start_tour("/web", 'sale_product_configurator_advanced_tour', login='salesman')
 
-        # Ensures some dynamic create variants have been created by the configurator
-        self.assertEqual(len(product_template.product_variant_ids), 2)
+        # Ensures dynamic create variants have been created by the configurator
+        self.assertEqual(len(product_template.product_variant_ids), 1)
         self.assertEqual(
-            len(product_template.product_variant_ids.product_template_attribute_value_ids),
-            8,
-            "2 variants are created during the tour, with each 5 PAV, but only 8 distinct PAV."
+            len(product_template.product_variant_ids.product_template_attribute_value_ids), 5
```

Comment:
This shouldn't change with this task, there shouldn't be a comportment change here. Keep creating 2 variants in the tours.

Reply:
Hm, this could be interesting indeed, but not required, indeed. Whatever you choose =]